### PR TITLE
Enforce Canvas to be LTR to support Moveable

### LINKED ISSUE #508: RTL Support for Drag & Drop (Movable)
## Feature Description

Follow-up to #55 / #351

When using WordPress admin in RTL context (e.g. by changing your locale or by using the RTL Tester plugin), the X axis is basically flipped for Movable.

If dragging an element 10px to the left, it will be dropped 10px to the right. You can't miss it when testing.

Proposed solution:

Force canvas to be LTR, so that dragging an element to the left actually moves it to the left, and not the opposite.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## Testing Instructions
- Turn on RTL Tester plugin or change your locale to RTL language.
- Add some elements, move them around and resize
- Verify that the elements move as expected
- Test that left and right arrow keys move the elements as expected
- Test that the text written in the edit mode is still working RTL. (Cursor is in the "beginning" of the text for LTR people)

---

Fixes #508

### CODE REVIEW

File: assets/src/edit-story/components/canvas/canvasLayout.js [hunk 1]
```diff
@@ -67,18 +67,20 @@ function CanvasLayout() {
...
-          <SelectionCanvas>
-            <DisplayLayer />
-            <FramesLayer />
-            <NavLayer />
-          </SelectionCanvas>
-          <EditLayer />
-        </CanvasElementDropzone>
-      </CanvasUploadDropTarget>
-    </Background>
+    <StyleSheetManager stylisPlugins={[]}>
```
Comment (barklund) [inline assets/src/edit-story/components/canvas/canvasLayout.js:74]: But... how does this even work? Why would this fix it? I don't understand!

Comment (barklund) [reply assets/src/edit-story/components/canvas/canvasLayout.js:74]: Something about disabling the default `stylis` plugins, that normally do some RTL magic? Anyway, neat that it works, but it should probably come with a comment, as this is _god-tier_ bug fixing ;)

**Comment by swissspidy** [comment]
Is there any way we can add Karma tests for this?

We'd need to dynamically change the text direction. This requires changing `isRTL` in the config, and probably also setting `dir="rtl"` on the root node.

So maybe something like this in `fixture.js` (like `setFlags`):

```js
/**
 * Update the config.
 *
 * For instance, to test in right-to-left mode, call `setConfig` before
 * calling the `render()` method:
 * ```
 * beforeEach(async () => {
 *   fixture = new Fixture();
 *   fixture.setConfig({isRTL: true});
 *   await fixture.render();
 * });
 * ```
 *
 * @param {Object} config Config.
 */
setConfig(config) {
  this._config = { ...this._config, ...config };
}
```

And then in the `Fixture.render()` method:

```js
root.setAttribute('dir', this._config.isRTL ? 'rtl' : 'ltr' );
```

This way we could write a simple test that verifies that the layout is flipped (by taking snapshot) and that the Moveable works as expected.

Could also be added for dashboard easily

File: assets/src/edit-story/components/canvas/karma/rtl.karma.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+ */
+import { Fixture } from '../../../karma';
+import { useStory } from '../../../app/story';
+
+describe('RTL support', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    fixture.setConfig({ isRTL: true });
```
Comment (swissspidy) [inline assets/src/edit-story/components/canvas/karma/rtl.karma.js:28]: Whoa so this actually worked? Sweet!

Comment (miina) [reply assets/src/edit-story/components/canvas/karma/rtl.karma.js:28]: Yep, it did, thanks! 🎉  Waiting to see the screenshots but locally all worked.

**Comment by miina** [comment]
Hmm, the screenshots display just a white screen for some reason 😕

**Comment by miina** [comment]
> Hmm, the screenshots display just a white screen for some reason 😕

Ah, should have used `await`, too 🤦

File: assets/src/edit-story/components/canvas/canvasLayout.js (hunk 1 above)
Comment (miina) [reply assets/src/edit-story/components/canvas/canvasLayout.js:74]: @barklund Done, PTAL

**Review by swissspidy — APPROVED** [review]
Sweet!

File: assets/src/edit-story/components/canvas/karma/rtl.karma.js (hunk 2 above)
Comment (barklund) [reply assets/src/edit-story/components/canvas/karma/rtl.karma.js:28]: This is pretty awesome! Cool screenshots!

**Review by barklund — APPROVED** [review]
Big thumbs-up from me 👍 !

**Comment by csossi** [comment]
Verified in QA

**Pull request merged**